spec/index.html

```diff
@@ -266,35 +271,18 @@ <h2>A Canonical form of N-Quads</h2>
       MUST NOT use the datatype IRI part of the <a href="#grammar-production-literal">literal</a>,
       and are represented using only <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>.
     </li>
-    <!--li><code><a href="#grammar-production-HEX">HEX</a></code> MUST use only uppercase letters (<code>[A-F]</code>).</li-->
-    <li>Characters MUST NOT be represented by <code><a href="#grammar-production-UCHAR">UCHAR</a></code>.</li>
+    <li><code><a href="#grammar-production-HEX">HEX</a></code> MUST use only uppercase letters (<code>[A-F]</code>).</li>
     <li>Within <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>,
       the characters 
       <code>U+0022</code>, <code>U+005C</code>, <code>U+000A</code>, <code>U+000D</code>
```

Comment:
```suggestion
      <code>U+0022</code>, <code>U+005C</code>, <code>U+000A</code>, <code>U+000D</code>,
      <code>U+0008</code>, <code>U+0009</code>, <code>U+000C</code>, <code>U+0027</code>
```
IIRC, it might be a good idea to add here the other four "ECHAR"s, i.e., `\b` (U+0008), `\t` (U+0009), `\f` (U+000C), `\'` (U+0027), otherwise how to encode them seems currently ambiguous.

Reply:
Thanks, I think I misunderstood the following text:
> characters between U+0000 and U+001F along with U+007F that are not explicitly encoded using ECHAR must be encoded as UCHAR

From your comment, I now understand that these four ECHARs, `\b` (U+0008), `\t` (U+0009), `\f` (U+000C), `\'` (U+0027), should be encoded or represented as follows. Is that correct?
- `\b` (U+0008), `\t` (U+0009), `\f` (U+000C) => encoded using UCHAR (i.e., `\u0008`, `\u0009`, `\u000C`)
- `\'` (U+0027) => represented natively in Unicode (i.e., `'`)

Reply:
Probably requires tweaking for the characters you mention, but yes, in that lower range, characters are either ECHAR or UCHAR. Some characters outside that range also must use ECHAR and 007F must use UCHAR. I’ll tweak the wording some more tomorrow.

Reply:
Added a commit to clarify this, and use the relevant character abbreviations which were originally intended and in fact in my own implementation.